src/graph/test/SchemaTest.cpp

```diff
@@ -34,14 +35,16 @@ TEST_F(SchemaTest, metaCommunication) {
         std::string query = "ADD HOSTS 127.0.0.1:1000, 127.0.0.1:1100";
         auto code = client->execute(query, resp);
         ASSERT_EQ(cpp2::ErrorCode::SUCCEEDED, code);
+        meta::TestUtils::registerHB(
+                network::NetworkUtils::toHosts("127.0.0.1:1000, 127.0.0.1:1100").value());
     }
     {
         cpp2::ExecutionResponse resp;
         std::string query = "SHOW HOSTS";
         client->execute(query, resp);
-        std::vector<uniform_tuple_t<std::string, 2>> expected{
-            {"127.0.0.1", "1000"},
-            {"127.0.0.1", "1100"},
+        std::vector<uniform_tuple_t<std::string, 3>> expected{
```

Comment:
We'd better test one case with host offline.

Reply:
roger that

Reply:
SchemaTest has other ut which need to use active hosts. I add ut in meta/test/ProcessorTest.